README.md

```diff
@@ -26,6 +26,7 @@ By using OAuth2, it ensures users can't forge or fake appeals.
    | Client ID             | You can get this from the **General Information** section for the application you created in step 1.                       |
    | Client secret         | You can get this from the **General Information** section for the application you created in step 1.                       |
    | Bot token             | Get this in the **Bot** section that you used in step 2.                                                                   |
+   | Webhook URL           | You can create it in Discord Channel Settings -> Integrations -> Webhooks                                                  |
```

Comment:
I would recommend removing this from the template, because if it's in the template people will by default be tempted to fill in this information, but may want the unban button.

Reply:
> I would recommend removing this from the template, because if it's in the template people will by default be tempted to fill in this information, but may want the unban button.

Yeah, i think more detailed readme section will be ok then.